webapp/channels/src/components/team_sidebar/components/team_button.tsx

```diff
@@ -192,10 +192,12 @@ export default function TeamButton({
                         ref={provided.innerRef}
                         {...provided.draggableProps}
                         {...provided.dragHandleProps}
+                        tabIndex={-1}
```

Comment:
Do we need two tab indices here?

Reply:
Yeah on closer inspection I don't. On testing, it seems only the draggable-team-container div had an issue with tabbing. I removed the extra tab indices on the team-container divs and it works the same.